R/incotrend.R

```diff
@@ -276,6 +276,7 @@ BEV,Electric
 NG,Liquids
 Hybrid Electric,Liquids")
     mimap <- fread(system.file("extdata", "mitigation-techmap.csv", package="edgeTransport"))
+    mimap <- mimap[!FV_vehvar == "LDV|4W"]
```

Comment:
seems legit. Did this lead to issues? SWs are zero for those no?

Reply:
No it was more a safety measure as I introduced these entries in the techmap and didn't have the time to track all data wrangling steps in incotrend (I am not so familiar with the notation you are using)